content/connect/index.textile

```diff
@@ -0,0 +1,263 @@
+
+h2(#heartbeats). Heartbeats
+
+Heartbeats enable Ably to identify clients that abruptly disconnect from the service, such as where an internet connection drops out or a client changes networks.
+
+Ably sends a heartbeat to connected clients every 15 seconds. If a client goes more than 25 seconds without seeing any server activity from Ably, it assumes that something has gone wrong with the connection and the "connection state":/connect/states will become @disconnected@. The 25 seconds the client waits is the heartbeat interval plus a 10 second margin of error to allow for network delays.
+
+Ably also uses this mechanism to detect dropped client connections, though some details vary depending on the transport used.
+
+It is important to note that this mechanism is only used when something disrupts communication and does not properly terminate the TCP connection. It isn't used when a connection is deliberately closed or disconnected, for example by calling the "@close()@ method":/api/realtime-sdk/connection#close or being disconnected by the server.
+
+The 15 second interval between heartbeats is used to strike a balance between optimizing battery usage for client devices and the time it takes to identify a dropped or unstable connection.
+
+The interval between heartbeats can be customized if your app requires increased battery preservation or to identify dropped connections more quickly. Set a value between 5000 and 1800000 milliseconds (5 seconds and 30 minutes) using the @heartbeatInterval@ parameter within the @transportParams@ property of the "@clientOptions@":/api/realtime-sdk#client-options object.
+
+Using a higher @heartbeatInterval@ can increase the time taken for the Ably service and the client itself to identify a connection has dropped when an abrupt disconnect occurs. The number of "peak connections":/account/LINK-ME may also appear higher as it can take longer to terminate dropped connections. Although @heartbeatInterval@ can be set as high as 30 minutes, Ably does not recommend setting it this high.
+
+The following example code demonstrates establishing a connection to Ably with a @heartbeatInterval@ of 10 seconds:
+
+bc[jsall]. const ably = new Ably.Realtime(
+  {
+    key: '{{API_KEY}}',
+    transportParams: { heartbeatInterval: 10000 }
+  }
+);
+
+h2(#closing-a-connection). Closing a connection
+
+p(note). There is a 2 minute delay if a connection is not explicitly closed which could impact peak connections and thus costs.
+
+blang[default,javascript].
+  A connection to Ably can be closed once it is no longer needed.
+
+  The following code sample explicitly closes the connection to Ably and prints the message @Closed the connection to Ably@.
+
+blang[php].
+  The <span lang="php">PHP</span> SDK is only available with a REST interface. You will need to use one of the realtime libraries to create or manage a connection to Ably. Use the language selector above to select a language with realtime support.
```

Comment:
PHP is REST-only so this can be removed as it won't be possible to select it as a language on the page.